numba/core/callconv.py

```diff
@@ -564,6 +564,8 @@ def emit_unwrap_dynamic_exception_fn(self, module, st_type, nb_types):
 
         fnty = ir.FunctionType(GENERIC_POINTER, [GENERIC_POINTER])
         fn = ir.Function(module, fnty, name)
+        # Linkage is changed to linkonce_odr for PYCC.
+        fn.linkage = "external"
```

Comment:
This line has no effect because "external" is the default. Just added so the comment has something to associate with.